src/harness/harnessLanguageService.ts

```diff
@@ -333,6 +333,9 @@ module Harness.LanguageService {
         getDefinitionAtPosition(fileName: string, position: number): ts.DefinitionInfo[] {
             return unwrapJSONCallResult(this.shim.getDefinitionAtPosition(fileName, position));
         }
+        getTypeDefinitionAtPosition(fileName: string, position: number): ts.DefinitionInfo[]{

```

Comment:
What is the difference between getDefinitionAtPositoin and getTypeDefinitionAtPosition?